omnigibson/utils/motion_planning_utils.py

```diff
@@ -0,0 +1,543 @@
+import numpy as np
+from math import ceil
+
+import omnigibson as og
+from omnigibson.object_states import ContactBodies
+import omnigibson.utils.transform_utils as T
+from omnigibson.utils.control_utils import IKSolver
+from pxr import PhysicsSchemaTools, Gf
+
```

Comment:
which repeated code are you referring to?

Reply:
A lot of this stuff _seems_ like repetition because they are the same components (state checker, sampling, etc) but they are actually entirely different for the different kinds of planning so I think this is OK for now. Hard to unify anything in this file.